lib/logsource.h

```diff
@@ -80,6 +80,7 @@ struct _LogSource
 
   void (*wakeup)(LogSource *s);
   void (*dynamic_window_realloc)(LogSource *s);
```

Comment:
This commit should remove this method in favor of `schedule_dynamic_window_realloc`.